src/compiler/statement/IfStatement.ts

```diff
@@ -0,0 +1,38 @@
+import * as ts from "typescript";
+import {removeStatementedNodeChild} from "./../../manipulation";
+import {Node, Expression} from "./../common";
+import {ChildOrderableNode} from "./../base";
+import {Statement} from "./Statement";
+
+export const IfStatementBase = ChildOrderableNode(Statement);
+export class IfStatement extends IfStatementBase<ts.IfStatement> {
+    /**
+     * Gets this if statement's expression.
+     */
+    getExpression() {
+        return this.global.compilerFactory.getNodeFromCompilerNode(this.compilerNode.expression, this.sourceFile) as Expression;
```

Comment:
Sidenote: I really need to just add an internal convenience method:

```ts
this.getNodeFromCompilerNode(this.compilerNode.expression);
```

I'll open an issue for that and then change the entire code base (opened #206)